packages/epilot-sdk/src/template-variable-client.ts

```diff
@@ -0,0 +1,6 @@
+import { getClient } from '@epilot/template-variables-client';
```

Comment:
Module name should match the package name.

`packages/epilot-sdk/src/template-variables-client.ts`